### DISCUSSION TITLE
Widgets - Add option to change font weight

### DISCUSSION THREAD

**Discussion by Barcaman1** [breezy-weather/breezy-weather#1431] [Ideas]
### Describe your idea

Is there any chance to add an option to change font of the text in widgets. For example here I think text would look much better with "Sans Serif Black" font (like in the clock) rather than default one.
![Screenshot_2024-11-17-13-34-07-47_8b90da7a90336dddbe1fd9fd7538077a](https://github.com/user-attachments/assets/2a4caf1c-1f1b-4b4e-a1be-73d1afd61f3a)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
No new features will be added to the legacy widgets.

As for the modern widgets, it's not planned but the font weight is already slightly bolder, so I'm not sure it's needed:
![image](https://github.com/user-attachments/assets/cd4fcd9c-d593-4d28-af1a-817d6a926f7a)
